CRM/Core/I18n.php

```diff
@@ -239,6 +239,26 @@ public static function languages($justEnabled = FALSE) {
     return $justEnabled ? $enabled : $all;
   }
 
+  /**
+   * Get the options available for format locale.
+   *
+   * Note the pseudoconstant can't be used as the key is the name not the value.
+   *
+   * @return array
+   */
+  public static function getLocaleOptions(): array {
+    $values = CRM_Core_OptionValue::getValues(['name' => 'languages'], $optionValues, 'label', TRUE);
+    foreach ($values as $value) {
+      $return[$value['name']] = $value['label'];
+    }
+    // Sorry not sorry.
+    // Hacking in for now since the is probably the most important use-case for
+    // money formatting in an English speaking non-US locale based on any reasonable
+    // metric.
+    $return[$value['en_NZ']] = ts('English - New Zealand');
```

Comment:
@eileenmcnaughton I think this should be just 'en_NZ' ?

Reply:
oh yeah